.rubocop_todo.yml

```diff
@@ -20,7 +20,7 @@ Metrics/AbcSize:
 Metrics/ClassLength:
   Max: 1300
 Metrics/ModuleLength:
-  Max: 1600
+  Max: 1700
```

Comment:
One day we will split the files. One day... Before that this number is going to keep increasing as we add more tests to the `base_test.rb` file.